File: app/src/main/java/com/mvrborges/projectgold/CreateNewAccount.java
package com.mvrborges.projectgold;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class CreateNewAccount extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.setTitle(getResources().getText(R.string.createLogin_title));
        setContentView(R.layout.activity_create_new_account);
    }

    public void onClickBtnCancel(View view) {
        Intent inCreate = new Intent(CreateNewAccount.this, Login.class);
        startActivity(inCreate);
    }

    public void onClickBtnCreate(View view) {
        Intent inCreate = new Intent(CreateNewAccount.this, Login.class);
        startActivity(inCreate);
    }
}

File: app/src/main/java/com/mvrborges/projectgold/configs/CreateNewAccount.kt
package com.mvrborges.projectgold.configs

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

import com.mvrborges.projectgold.R

class CreateNewAccount : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        this.title = resources.getText(R.string.createLogin_title)
        setContentView(R.layout.activity_create_new_account)
    }

    fun onClickBtnCancel(view: View) {
        val inCreate = Intent(this@CreateNewAccount, Login::class.java)
        startActivity(inCreate)
    }

    fun onClickBtnCreate(view: View) {
        val inCreate = Intent(this@CreateNewAccount, Login::class.java)
        startActivity(inCreate)
    }
}
